### ISSUE TITLE
Grid View/ Image search

### ISSUE THREAD

**Issue by ch-sander** [lombardpress/lombardpress-webjs#117]
I think we've got this started with commit 4c29a862608d794102df0ecbcc1dab256ef96051

It can definitely be improved with small labels and lazy loading. But I think we've got the grid/table part started.

_Originally posted by @jeffreycwitt in https://github.com/lombardpress/lombardpress-webjs/issues/97#issuecomment-1135262513_

**Comment by ch-sander** [comment]
Yes, I think so! Yes, small labels, would be ideal (page number if only one manifestation and/or logical position). And maybe make it sortable and subject to a filter by structure (just "book 1")?

**Comment by jeffreycwitt** [comment]
I think commit https://github.com/lombardpress/lombardpress-webjs/commit/25df3eff1738719d4459eca846614c14fd8d8ff0 makes first step toward resolving this. It maybe not be the perfect or final solution, but I think it works for now. (See gif below)

![imageSearchResults](https://user-images.githubusercontent.com/1146685/185925907-c68b216a-96e2-4a8c-b3e5-1aefe3e15fec.gif)
